src/supertux/sector.hpp

```diff
@@ -267,6 +267,11 @@ class Sector final : public Base::Sector
   float m_last_scale;
   float m_last_dt;
 
+ public: 
+  // The default sector size.
+  static const int DEFAULT_SECTOR_W = 350;
```

Comment:
Can you name these members `DEFAULT_SECTOR_WIDTH` and `DEFAULT_SECTOR_HEIGHT` please?